Essentials/src/messages.properties

```diff
@@ -71,6 +71,7 @@ compassBearing=\u00a76Bearing\: {0} ({1} degrees).
 configFileMoveError=Failed to move config.yml to backup location.
 configFileRenameError=Failed to rename temp file to config.yml.
 confirmClear=\u00a77To \u00a7lCONFIRM\u00a77 inventory clear, please repeat command: \u00a76{0}
+confirmCommandPayment=\u00a76Perform this command again to confirm this command payment.
```

Comment:
This message should include the command that is being charged for, along with the amount that the player would be charged.